drivers/can/Kconfig.mcp2515

```diff
@@ -0,0 +1,141 @@
+# Kconfig.mcp2515 - MCP2515 CAN configuration options
+
+#
+# Copyright (c) 2018 Karsten Koenig
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config CAN_MCP2515
+	bool "MCP2515 CAN Driver"
+	default n
+	depends on SPI
+	help
+	  Enable MCP2515 CAN Driver
+
+if CAN_MCP2515
+
+config CAN_MCP2515_NAME
+	string "Driver name"
+	default "MCP2515"
+
+config CAN_MCP2515_OSC_FREQ
+	int "Oscillator frequency"
+	default 8000000
+	help
+	 Specify the frequency of the oscillator connected to the MCP2515.
```

Comment:
Nitty, but help texts are usually indented by two spaces instead of one.

Reply:
Fixed, must have slipped :-)